api/python/notebooks/api_demo/census_duplicated_cells.ipynb

```diff
@@ -0,0 +1,464 @@
+   "source": [
+    "import cellxgene_census\n",
+    "\n",
+    "tabula_muris_dataset_id = \"48b37086-25f7-4ecd-be66-f5bb378e3aea\"\n",
+    "\n",
+    "with cellxgene_census.open_soma() as census:\n",
+    "    tabula_muris_obs = census[\"census_data\"][\"mus_musculus\"].obs.read(\n",
+    "        value_filter=f\"dataset_id == '{tabula_muris_dataset_id}'\", column_names=[\"tissue\", \"is_primary_data\"]\n",
+    "    )\n",
+    "\n",
+    "    tabula_muris_obs = tabula_muris_obs.concat().to_pandas()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's take a look at counts for the unique combinations of values.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "tissue           is_primary_data  dataset_id                          \n",
+       "bone marrow      True             48b37086-25f7-4ecd-be66-f5bb378e3aea    40220\n",
+       "spleen           True             48b37086-25f7-4ecd-be66-f5bb378e3aea    35718\n",
+       "limb muscle      True             48b37086-25f7-4ecd-be66-f5bb378e3aea    28867\n",
+       "lung             True             48b37086-25f7-4ecd-be66-f5bb378e3aea    24540\n",
+       "kidney           True             48b37086-25f7-4ecd-be66-f5bb378e3aea    21647\n",
+       "tongue           True             48b37086-25f7-4ecd-be66-f5bb378e3aea    20680\n",
+       "mammary gland    True             48b37086-25f7-4ecd-be66-f5bb378e3aea    12295\n",
+       "thymus           True             48b37086-25f7-4ecd-be66-f5bb378e3aea     9275\n",
+       "bladder lumen    True             48b37086-25f7-4ecd-be66-f5bb378e3aea     8945\n",
+       "heart            True             48b37086-25f7-4ecd-be66-f5bb378e3aea     8613\n",
+       "trachea          True             48b37086-25f7-4ecd-be66-f5bb378e3aea     7976\n",
+       "liver            True             48b37086-25f7-4ecd-be66-f5bb378e3aea     7294\n",
+       "adipose tissue   True             48b37086-25f7-4ecd-be66-f5bb378e3aea     6777\n",
+       "pancreas         True             48b37086-25f7-4ecd-be66-f5bb378e3aea     6201\n",
+       "skin of body     True             48b37086-25f7-4ecd-be66-f5bb378e3aea     4454\n",
+       "large intestine  True             48b37086-25f7-4ecd-be66-f5bb378e3aea     1887\n",
+       "Name: count, dtype: int64"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "tabula_muris_obs.value_counts()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can see all cells across the tissues are labelled as `True` for `is_primary_data`.\n",
+    "\n",
+    "But what if we select cells from the dataset that only contains cells from the liver: \"Liver - A single-cell transcriptomic atlas characterizes ageing tissues in the mouse - 10x\".\n"
```

Comment:
```suggestion
    "But now let's select cells from a dataset that was _derived_ from Tabula Muris Senis dataset and only include its liver cells: \"Liver - A single-cell transcriptomic atlas characterizes ageing tissues in the mouse - 10x\"?\n"
```
Just trying to better establish the relationship between the primary and non-primary datasets, to aid the reader.